dev/create-release/releaseutils.py

```diff
@@ -49,6 +49,9 @@
     print("Install using 'sudo pip install unidecode'")
     sys.exit(-1)
 
+if sys.version_info[0] >= 3:
+    raw_input = input
```

Comment:
Does this script work in Python 3+ now?

Reply:
It creates a new function called __raw_input()__ that is identical to the builtin __input()__.